t/plugin/traffic-split.t

```diff
@@ -1972,28 +1964,27 @@ passed
 
 
 
-=== TEST 54: /not_found
---- request
-GET /not_found
---- error_code: 404
---- response_body
-{"error_msg":"404 Route Not Found"}
---- no_error_log
-[error]
---- LAST
-
+=== TEST 54: hit routes
+--- config
+location /t {
+    content_by_lua_block {
+        local t = require("lib.test_admin").test
+        do
+            local code, _ = t('/hello', ngx.HTTP_GET)
+        end
 
-=== TEST 55: hit routes
---- pipelined_requests eval
-["GET /server_port", "GET /server_port"]
---- error_code eval
-[200, 502]
+        local code, _ = t('/hello', ngx.HTTP_GET)
+        if code >= 300 then
+            ngx.status = code
+        end
+    }
+}
+--- request
+GET /t
+--- error_code: 502
 --- grep_error_log eval
 qr/\[error\]/
 --- grep_error_log_out
 [error]
```

Comment:
```
[error]
[error]
[error]
```

this is not a good way to check the error log. we need to confirm the keyword

Reply:
updated.